service/src/main/kotlin/fi/espoo/evaka/invoicing/controller/PaymentController.kt

```diff
@@ -19,21 +19,19 @@ import fi.espoo.evaka.shared.domain.EvakaClock
 import fi.espoo.evaka.shared.security.AccessControl
 import fi.espoo.evaka.shared.security.Action
 import java.time.LocalDate
+import mu.KotlinLogging
 import org.springframework.web.bind.annotation.PostMapping
 import org.springframework.web.bind.annotation.RequestBody
-import org.springframework.web.bind.annotation.RequestMapping
 import org.springframework.web.bind.annotation.RestController
 
+private val logger = KotlinLogging.logger {}
+
 @RestController
-@RequestMapping(
-    "/payments", // deprecated
-    "/employee/payments"
-)
 class PaymentController(
     private val accessControl: AccessControl,
     private val paymentService: PaymentService
 ) {
-    @PostMapping("/search")
+    @PostMapping("/payments/search")
```

Comment:
These old endpoints should have *both* the "/payments" and "/employee/payments" prefixes until we finish migrating to the latter